cfme/configure/tasks.py

```diff
@@ -36,12 +39,17 @@
     ]
 )
 
+table_loc = '//div[@id="records_div"]/table'
```

Comment:
I'll try that - I actually had it there first but moved it into the tabs because I was worried about the table getting set when the parent gets instantiated and isn't on the right tab yet.